src/components/MyKiva/BlogCard.vue

```diff
@@ -66,6 +66,22 @@ const props = defineProps({
 	},
 });
 
+const optimizedImageUrl = computed(() => {
+	if (!props.image) return '';
+
+	// blog card dimensions: w:336px, h:55% = ~185px
```

Comment:
Are we sure about these dimensions? The images seem to be 336x200 for me

<img width="393" height="479" alt="Image" src="https://github.com/user-attachments/assets/a4eb69fe-e0fc-4122-b78d-84a76fee7655" />